Packs/Darktrace/Integrations/Darktrace/Darktrace.yml

```diff
@@ -478,7 +478,45 @@ script:
         keys. It is recommended to run the command once to check the relevant outputs
         in context.
       type: Unknown
-  dockerimage: demisto/python3:3.9.5.21272
+  - name: darktrace-get-breach-details
+    arguments:
+    - name: pbid
+      required: true
+      description: Darktrace modelbreach ID
+    - name: endtime
+      description: End time of data to return in millisecond format, relative to midnight January 1st 1970 UTC
+    - name: count
+      defaultValue: "100"
+      description: Specifies the maximum number of items to return. Default 100
+    - name: offset
+      defaultValue: "0"
+      description: Offset of starting point for fetching details of the modelbreach
+    description: Retrieve additional details on a modelbreach
+    outputs:
+    - contextPath: Darktrace.ModelBreach
+      description: ''
+      type: Unknown  
```

Comment:
Please fill in the description and add the type of the returned value. Also in the following outputs.